bot/seasons/evergreen/minesweeper.py

```diff
@@ -0,0 +1,266 @@
+import logging
+import typing
+from dataclasses import dataclass
+from random import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Client
+
+MESSAGE_MAPPING = {
+    0: ":stop_button:",
+    1: ":one:",
+    2: ":two:",
+    3: ":three:",
+    4: ":four:",
+    5: ":five:",
+    6: ":six:",
+    7: ":seven:",
+    8: ":eight:",
+    9: ":nine:",
+    10: ":keycap_ten:",
+    "bomb": ":bomb:",
+    "hidden": ":grey_question:",
+    "flag": ":triangular_flag_on_post:"
+}
+
+log = logging.getLogger(__name__)
+
+
+class CoordinateConverter(commands.Converter):
+    """Converter for Coordinates."""
+
+    async def convert(self, ctx, coordinate: str) -> typing.Tuple[int, int]:
+        """Take in a coordinate string and turn it into x, y"""
+        if not 2 <= len(coordinate) <= 3:
+            raise commands.BadArgument('Invalid co-ordinate provided')
+
+        digit, letter = sorted(coordinate.lower())
+
+        if not letter.isdigit():
```

Comment:
This is probably the fix

```suggestion
        if not digit.isdigit():
```

Reply:
Yes, I think so. Thanks for that. On my phone so could not fix it myself